modules/olm-bundle-format.adoc

```diff
@@ -40,7 +39,16 @@ deployment and RBAC model of the Operator.
 
 A bundle includes one ClusterServiceVersion (CSV) per directory
 and typically the CRDs that define the owned APIs of the CSV in its `manifest/`
-directory, though additional objects may be included:
+directory.
```

Comment:
nit: within OLM we generally refer to the directory as `/manifests` to indicate it is at the root level. Also it is plural, not singular.